### PR TITLE
adding static flower pattern

### PR DESCRIPTION
**Pull request by umjennifer**
adding static flower pattern

### CODE REVIEW

File: fun.py [hunk 1]
```diff
@@ -172,6 +172,25 @@ def off(self):
...
+        while True:
+            #variable for yellow  petals
+            petal_color = (255,233,0)
+            #set background to purple
+            pixels = [(149,44,201)] * numLEDs
+            for j in patt['flower']['pistil']:
+                #set pistil to brown
+                pixels[j] = (66,28,19)
+            for j in patt['flower']['left']:
+                pixels[j] = (petal_color)
```
Comment (jlchamaa) [inline fun.py]: petal_color is already a tuple, this will make it a tuple of a tuple

File: patterns.py [hunk 2]
```diff
@@ -56,6 +56,14 @@
...
     'david':[11,19,20,21,22,23,34,33,32,31,30,42],
 
+    'flower':{
+            'pistil':[20,21,22,31,32,33],
+            #petals
+            'left':[24,25,26,27,28,29],
+            'top':[41,42,43,49,50,51],
+            'right':[16,17,18,35,36,37],
+            'bottom':[2,3,4,10,11,12]
+            },
```
Comment (jlchamaa) [inline patterns.py:66]: nitpick: add a line of space after this to stay consistent with the other entries

**Review by jlchamaa — CHANGES_REQUESTED** [review]
fix the tuples in fun.py:flower()

File: fun.py (hunk 1 above)
Comment (umjennifer) [reply fun.py]: I see because it would be ((255,233,0))

**Review by jlchamaa — APPROVED** [review]
LGTM

**Pull request merged**